src/gaggiuino.ino

```diff
@@ -228,12 +227,12 @@ static void pageValuesRefresh(bool forcedUpdate) {  // Refreshing our values on
     runningCfg.pressureProfilingHold          = myNex.readNumber("ppHold"); // pp start pressure hold
     runningCfg.pressureProfilingLength        = myNex.readNumber("ppLength"); // pp shot length
 
-    runningCfg.flowProfileStart               = myNex.readNumber("ppFlowStart") / 10.f;
-    runningCfg.flowProfileEnd                 = myNex.readNumber("ppFlowFinish") / 10.f;
+    runningCfg.flowProfileStart               = myNex.readNumber("ppFlowStart") / 10;
+    runningCfg.flowProfileEnd                 = myNex.readNumber("ppFlowFinish") / 10;
```

Comment:
If you remove the float from here wouldn't it do an integer division? (which means it would truncate the decimal part)

Reply:
Shit you right was an omission really